example-scripts/voctolight/voctolight.py

```diff
@@ -83,14 +63,14 @@ def start_connection(tally_handler):
         except IndexError:
             pass
 
-
-if __name__ in '__main__':
+def main():
+    plugin_cls = PLUGINS.get(Config.get('light', 'plugin'), None)
```

Comment:
Instead of replicating the code from `testlight.py`, re-use the aforementioned `get_plugin_cls_by_name` function to centralize this code snippet.

Reply:
Done.